airwallex/src/main/java/com/airwallex/android/view/PaymentMethodsActivity.kt

```diff
@@ -56,17 +56,27 @@ class PaymentMethodsActivity : AirwallexCheckoutBaseActivity() {
     }
 
     private var availablePaymentMethodTypes: List<AvailablePaymentMethodType>? = null
+    private var availablePaymentConsents: List<PaymentConsent>? = null
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
         fetchPaymentMethods()
     }
 
-    private fun initView(availablePaymentMethodTypes: List<AvailablePaymentMethodType>) {
+    override fun onBackPressed() {
+        super.onBackPressed()
+        setResult(RESULT_CANCELED)
```

Comment:
I think even without this, its parent activity `PaymentCartFragment` will receive `RESULT_CANCELED` resultCode.

Reply:
Are you referring to what was in `AirwallexActivity`? I needed to remove that because it made it impossible to override the result when pressing back.